test/VisionModel/googlenet.jl

```diff
@@ -0,0 +1,11 @@
+using Flux
+using FluxModels
+using Test
```

Comment:
Can you remove this from all the files? It is only necessary in `runtests.jl`.

Reply:
ok, now will have to test vgg models.